docs/platform.rst

```diff
@@ -1,22 +1,51 @@
+.. _platform:
+
 Platform
 ========
 
-contains platform details
+OpenQL supports various target platforms.
+These platforms can be software simulators or architectures targetting hardware quantum computers.
+The following platforms are supported by OpenQL:
+
+- software simulator platforms
+	- :ref:`qxplatform`
+	- :ref:`qsimplatform`
+- hardware platforms
+	- :ref:`cclplatform`
+	- :ref:`ccplatform`
+	- :ref:`cboxplatform`
+
+
+A platform can be created in OpenQL by using ``Platform()`` API as show below:
```

Comment:
I think it should be made explicit that a platform can only be created based on a configuration file (that is described in the section below). 

Because this refers to the API one is triggered to look for instructions that create a platform step by step whereas the creation is done based on what is defined int he configuration file.